experimental/jackPython/astMapping/main.py

```diff
@@ -116,11 +130,11 @@ def translator(input_filename, ds, inputLanguage, outputLanguage):
         return
 
     for node in gast["body"]:
-        print(astToJsMap[node["type"]][outputLanguage])
+        print(ds[node["type"]][outputLanguage])
 
 
 #TODO: replace the mapping for each one with a function that returns the correct output string WITH arguments
-astToJsMap = {
+ds = {
     "logStatement": {
         "py": "print()",
         "js": "console.log()"
```

Comment:
IMO the python converter one should not have logic anything related to JS nor should the gast.

Reply:
I just separated py_to_gast into its own separate file in order to solidify this distinction